
/etc/ldap.con is not OpenLDAP - specific. It is also used by pam_ldap, nss_ldap and 
propably other software

diff -durN ldap.orig/include/ldapconfig.h.edit ldap/include/ldapconfig.h.edit
--- ldap.orig/include/ldapconfig.h.edit	Sat Jan 23 22:24:59 1999
+++ ldap/include/ldapconfig.h.edit	Fri Sep 10 14:19:09 1999
@@ -39,7 +39,7 @@
 #define DEFAULT_LIBEXECDIR	"%LIBEXECDIR%"
 #define DEFAULT_RUNDIR		"%RUNDIR%"
 
-#define DEFAULT_LDAP_CONF_FILE "%SYSCONFDIR%/ldap.conf"
+#define DEFAULT_LDAP_CONF_FILE "/etc/ldap.conf"
 #define DEFAULT_LDAP_USERRC_FILE "ldaprc"
 #define DEFAULT_LDAP_ENV_PREFIX "LDAP"
 
